client/components/Editor/styles/suggestedEdits.scss

```diff
@@ -43,7 +43,12 @@ span[data-suggestion-mark='true'][data-suggestion-kind='addition'] {
 }
 
 span[data-suggestion-mark='true'][data-suggestion-kind='removal'] {
-	text-decoration: 2px $removal line-through;
+	text-decoration: 2px;
+	-webkit-text-decoration: 2px;
```

Comment:
safari doesn't seem to recognize this property, so the line is 1px. however, it looks like there's wide support for `text-decoration-thickness`, including unprefixed on safari

Reply:
oof, thought i copied from the stash on master